### PR TITLE
CIDEVSTC-277 - Adds RetrieveProcess a simple interface to retrieve

### PR DESCRIPTION
**Pull request by lukecampbell**
[CIDEVSTC-277](https://jira.oceanobservatories.org/tasks/browse/CIDEVSTC-277)

### CODE REVIEW

File: ion/processes/data/replay/replay_process.py [hunk 1]
```diff
@@ -306,5 +309,31 @@ def _replay(self):
...
+        '''
+        coverage = self.get_coverage()
+        # Convert python datetimes to unix timestamps
+        if isinstance(time1, datetime):
+            time1 = calendar.timegm(time1.timetuple())
+        if isinstance(time2, datetime):
+            time2 = calendar.timegm(time2.timetuple())
+
+        rdt = ReplayProcess._cov2granule(coverage, time1, time2)
+        return rdt
```
Comment (lukecampbell) [inline ion/processes/data/replay/replay_process.py:332]: This is what you can use. It's super simple with no or very few edge cases.

**Comment by lukecampbell** [comment]
@MauriceManning ready for review.

**Comment by lukecampbell** [comment]
Here's the test coi_any
http://buildbot.oceanobservatories.org:8010/builders/coi_any/builds/158
